web/source/dom/domEventHandlers.ts

```diff
@@ -206,7 +206,9 @@ namespace com.keyman.dom {
 
       // Makes sure we properly detect the TouchAliasElement root, 
       // rather than one of its constituent children.
-      Ltarg = findTouchAliasTarget(Ltarg) || Ltarg;
+      if(this.keyman.util.device.touchable) {
+        Ltarg = findTouchAliasTarget(Ltarg);
+      }
```

Comment:
While this block doesn't seem to be affected... _currently_... it makes the same assumptions that lead to the error we do see.